app/components/user-session/UserSession.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+    'use strict';
+
+    var module = angular.module('chatApp.UserSession', ['chatApp.DataModel']);
+        
+    module.controller('LoginController', function($scope, DataModelService) {
+            $scope.isLoggedIn = function() {
+                return DataModelService.isLoggedIn();
+            };
+            $scope.userName = '';
+            $scope.password = '';
+            $scope.login = function() {
+                DataModelService.setActiveUser($scope.userName, $scope.password);
+                $scope.userName = '';
```

Comment:
it is better to move this logic into separate function
the code will look better